package.json

```diff
@@ -62,7 +63,7 @@
     "express-rate-limit": "^5.3.0",
     "express-session": "^1.17.2",
     "file-saver": "^2.0.5",
-    "file-type": "^16.5.3",
+    "file-type": "^16.5.4",
```

Comment:
for this, would we need to bump the version of `file-type` in this PR? similarly for `commitizen`